Add path variable which can be used to set a path for all supported IAM resources (#37)
schubergphilis/terraform-aws-mcaf-avm: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).

> # 2.6.0 (2021-01-20)

Reviewer: ```suggestion
## 2.5.1 (2023-01-23)
```

> 

Reviewer: ```suggestion
BUG FIXES

```

- Add path variable to the permissions_boundary (workspace_boundary and workload_boundary)  ([#37] https://github.com/schubergphilis/terraform-aws-mcaf-avm/pull/37))

> # 2.5.0 (2021-01-20)

Reviewer: ```suggestion
## 2.5.0 (2023-01-20)
```

The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).

# 2.6.0 (2023-01-24)

> - Add path variable to the permissions_boundary (workspace_boundary and workload_boundary)  ([#37] https://github.com/schubergphilis/terraform-aws-mcaf-avm/pull/37))

Reviewer: ```suggestion
- Add path variable which can be used to set a path for all supported IAM resources  ([#37] https://github.com/schubergphilis/terraform-aws-mcaf-avm/pull/37))
```
